bootstrap/signer/main/main.go

```diff
@@ -10,37 +10,35 @@ import (
 	"context"
 	"flag"
 	"fmt"
+	"os"
+
+	"github.com/orbs-network/scribe/log"
 	"github.com/orbs-network/signer-service/bootstrap/signer"
 	"github.com/orbs-network/signer-service/config"
-	"github.com/orbs-network/scribe/log"
-	"os"
 )
 
 func getLogger() log.Logger {
 	return log.GetLogger().WithOutput(log.NewFormattingOutput(os.Stdout, log.NewJsonFormatter()))
 }
 
 func main() {
-	httpAddress := flag.String("listen", ":7777", "ip address and port for http server")
+	httpAddressFlag := flag.String("listen", ":7777", "ip address and port for http server")
```

Comment:
when we will refactor all services, they will only listen on port 80 internally in the container, so they can be referenced by a service name only from a sibling container

Reply:
Yes, I'm not sure we will need this flag at all in v3.
For the purposes of this PR where we want to change as little as possible, are you OK with the current changes though?